src/Components/GameScreenComponents/Column.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import "../../styles/GameScreen.css"
+
+interface myProps{
+    columnNumber: string
+}
+
+function Column(props: myProps){
+    return(
+        <div className="Column">
+            <div className="Column-element"></div>
```

Comment:
This is another place where flexibility could be added using the `NUM_ROWS` constant mentioned in another comment